src/org/scada_lts/web/mvc/api/PointValueAPI.java

```diff
@@ -322,7 +322,6 @@ public class PointValueAPI {
     private static final Log LOG = LogFactory.getLog(PointValueAPI.class);
 
     private DataPointService dataPointService = new DataPointService();
-    private DataSourceService dataSourceService = new DataSourceService();
```

Comment:
There should be no changes here, it must remain as it was, i.e. we do not remove this declaration, it has already been corrected in another issue, it will lead to conflicts. :)